web-api/migration-terraform/main/lambdas/migrations/0021-practitioner-search-upper-case.js

```diff
@@ -0,0 +1,18 @@
+const migrateItems = async items => {
+  const itemsAfter = [];
+  for (const item of items) {
+    if (
+      (item.pk.startsWith('privatePractitioner|') ||
+        item.pk.startsWith('irsPractitioner|')) &&
```

Comment:
should these also include `inactivePractitioner` like the migration that follows?

Reply:
... we will rewrite these records whenever we update a practitioner (transitioning them from inactive => active)